packages/front-end/components/Tags/SortedTags.tsx

```diff
@@ -3,9 +3,10 @@ import Tag from "./Tag";
 
 export interface Props {
   tags?: string[];
+  color?: string;
 }
 
-export default function SortedTags({ tags }: Props) {
+export default function SortedTags({ tags, color }: Props) {
```

Comment:
People have been using colors for tags pretty heavily to group related ones together (e.g. all site sections are green, user types are red).  Might be better to not add this prop and always render the actual tag colors.

Reply:
Oh, ok. I saw you added bg and text color props to the modal so thought you changed the background to white.

Reply:
I would switch the metrics tooltip to have a white background. That way we can use the actual tag colors.